build/tools/utils.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+export PS4='\e[36m+ ${FUNCNAME:-main}@${BASH_SOURCE}:${LINENO} \e[0m'
```

Comment:
I see that we added this here, but in general, We should add it to each bash file, as we do with `set -x`

Reply:
Done